### PR TITLE
Adman Adapter: add transactionID

### PR DESCRIPTION
**Pull request by SmartyAdman**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

### CODE REVIEW

File: modules/admanBidAdapter.js [hunk 1]
```diff
@@ -109,6 +109,10 @@ export const spec = {
         bidFloor: getBidFloor(bid)
       }
 
+      if (bid.transactionID) {
+        placement.transactionID = bid.transactionID;
```
[the remark this answers is not in the document]
Comment (xmgiddev) [reply modules/admanBidAdapter.js]: yes, we pass it to dsps. so you mean put it like that?

[the remark this answers is not in the document]
Comment (SmartyAdman) [reply modules/admanBidAdapter.js]: yes, we pass it to dsps. did you mean to put it like that?

[the remark this answers is not in the document]
Comment (SmartyAdman) [reply modules/admanBidAdapter.js]: as I understand, transactionId should be correct, not ID

**Comment by patmmccann** [comment]
thanks! please cover with a test

**Comment by SmartyAdman** [comment]
> thanks! please cover with a test

done

**Review by patmmccann — APPROVED** [review]

**Pull request merged**